docs/releases/1.11.txt

```diff
@@ -179,6 +179,8 @@ Management Commands
 * The new :option:`loaddata --exclude` option allows excluding models and apps
   while loading data from fixtures.
 
+* The :djadmin:`runserver` Web server supports HTTP 1.1.
```

Comment:
I don't think it's specific to any test. The first request to the admin login page handles at "Waiting for localhost".